legacy/src/api/merchants/merchants.md

```diff
@@ -271,6 +271,52 @@ Returns a [paginated][] list of Merchants attached to an Account.
 }
 {% endjson %}
 
+### Search Merchants
+
+{% reqspec %}
+  GET '/api/merchants/search'
+  auth 'api-key'
+  query_param 'origin', '123.42,32.22'
+  query_param 'distance', '100'
+  query_param 'name', 'Store'
+{% endreqspec %}
+
+{% h4 Request Query Parameters %}
+
+|       Query Parameter   |           Type            |                                 Description                                                         |
+| :---------------------- | :------------------------ | :--------------------------------------------------------------------------------------------------- |
+| {% break _ origin %}    | {% dt String %} {% opt %} | The point to be searched around, formatted as a latitude,longitude                                   |
+| {% break _ distance %}  | {% dt Number %} {% opt %} | The distance from the origin to be searched around in km                                             |
+| {% break _ assetType %} | {% dt String %} {% opt %} | The assetType to filter merchants by. This can be supplied multiple times to further filter results  |
```

Comment:
You should add a link on the `assetType` text which links it to the assetTypes page.